### PR TITLE
All gene id to symbol map

### PR DESCRIPTION
**Pull request by pnrobinson**
This seems to work for most Orpha entries, however, some do not have MIM ids and get missed. It might be best to live with this as the error is less severe than missing the tRNA genes. We may want another workaround though.

### CODE REVIEW

File: phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/mgsa/MgsaLocalTest.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        finalGenes.add(gene);
+        i++;
+      }
+      if (i>M) break;
+    }
+
+    return ImmutableSet.copyOf(finalGenes);
+  }
+
+/*
```
Comment (iimpulse) [inline phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/mgsa/MgsaLocalTest.java]: Remove comment if obsolete

File: phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/Gene2DiseaseAssociationParser.java [hunk 2]
```diff
@@ -97,26 +104,22 @@ private void parseMim2geneAndGeneInfo(File homoSapiensGeneInfoFile, File mim2gen
...
-      System.err.println(e.toString());
+  private void parseOrphaToGene(File orphaToGeneFile, File mim2geneMedgenFile) {
+    OrphaGeneToDiseaseParser parser = new OrphaGeneToDiseaseParser(orphaToGeneFile, mim2geneMedgenFile);
+    Multimap<TermId, Gene> orphaToGene = parser.getOrphaDiseaseToGeneSymbolMap();
+    int size_before = associationMap.size();
+    for (Map.Entry<TermId, Gene> entry : orphaToGene.entries()) {
+      TermId orpha = entry.getKey();
+      Gene gene = entry.getValue();
+      GeneToAssociation g2a = new GeneToAssociation(gene, AssociationType.UNKNOWN);
+      //if (!associationMap.containsEntry(orpha, g2a)) {
```
Comment (iimpulse) [inline phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/Gene2DiseaseAssociationParser.java]: remove comment here if obsolete

File: phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/Gene2DiseaseAssociationParser.java [hunk 3]
```diff
@@ -127,7 +130,7 @@ private void parseOrphaToGene(File orphaToGeneFile) {
    * @throws IOException if the file cannot be read
    */
   private void parseGeneInfo(File homoSapiensGeneInfoFile) throws IOException {
-    ImmutableBiMap.Builder<TermId,String> builder=new ImmutableBiMap.Builder<>();
+    ImmutableMap.Builder<TermId,String> builder=new ImmutableMap.Builder<>();
```
Comment (iimpulse) [inline phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/Gene2DiseaseAssociationParser.java]: use intellij to format code

File: phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/assoc/OrphaGeneToDiseaseParserTest.java [hunk 4]
```diff
@@ -43,64 +43,82 @@
...
+   // assertTrue(genes.contains("SCN5A"));
+   // assertTrue(genes.contains("HCN4"));
+   // assertTrue(genes.contains("MYH6"));
+    assertTrue(genes.stream().map(Gene::getSymbol).filter(s -> s.equals("SCN5A")).findFirst().isPresent());
+    assertTrue(genes.stream().map(Gene::getSymbol).filter(s -> s.equals("HCN4")).findFirst().isPresent());
+    assertTrue(genes.stream().map(Gene::getSymbol).filter(s -> s.equals("MYH6")).findFirst().isPresent());
   }
 
+
+/*
```
Comment (iimpulse) [inline phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/assoc/OrphaGeneToDiseaseParserTest.java]: Same here.

File: phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/mgsa/MgsaLocalTest.java (hunk 1 above)
Comment (pnrobinson) [reply phenol-analysis/src/test/java/org/monarchinitiative/phenol/analysis/mgsa/MgsaLocalTest.java]: This is still WIP

**Pull request merged**